viper/src/main/java/com/w3engineers/ext/viper/util/lib/mesh/MeshProvider.java

```diff
@@ -42,13 +45,13 @@
     private String NETWORK_PREFIX = "telemesh_t1-";
     private final String SOCKET_URL = "https://multiverse.w3engineers.com/";
```

Comment:
multiverse url is static present here

Reply:
As per our new system we update this. And use it from travis environment variables.